axelor-account/src/main/java/com/axelor/apps/account/service/reconcile/ReconcileCheckServiceImpl.java

```diff
@@ -207,4 +203,21 @@ protected boolean isMissingTax(MoveLine it) {
         && moveLineToolService.isMoveLineTaxAccount(it)
         && it.getAccount().getIsTaxAuthorizedOnMoveLine();
   }
+
+  protected boolean checkMoveLineAmount(
+      Reconcile reconcile, MoveLine moveLine, MoveLine otherMoveLine, boolean isDebit)
```

Comment:
Please remove the boolean. You could pass `getDebit()` or `getCredit()` easily as an argument, or even simpler, use `moveLine.getDebit().add(moveLine.getCredit());`